scene/gui/rich_text_label.cpp

```diff
@@ -410,6 +412,18 @@ int RichTextLabel::_process_line(ItemFrame *p_frame, const Vector2 &p_ofs, int &
 							break; //don't allow lines longer than assigned width
 						}
 
+						// For info about the unicode range, see Label::regenerate_word_cache.
+						const CharType current = c[end];
+						const bool separatable = (current >= 0x2E08 && current <= 0xFAFF) || (current >= 0xFE30 && current <= 0xFE4F);
```

Comment:
Range probably should start with `2E80` not `2E08` and seems to be too broad (includes Cyrillic, Devanagari extended and other scripts that should not be separable).

Something like `2E80 - 9FCB`, `AC00 - D7FB`, `F900 - FAD9`, `FE30 - FE4F`, `20000 - 2FA1D` (The last range won't work on Windows, but should be supported on any other OS).

Reply:
Yeah, I noticed that. Should I change the range used in `Label` too?

Reply:
Yeah that would make sense.